### PR TITLE
VOTE-332: translation setup

### PR DESCRIPTION
**Pull request by SamiraMSadat**
Jira ticket: https://cm-jira.usa.gov/browse/VOTE-332

This PR adds the functionality to load different json files based on language code. 
1. New folder structure in /data
2. Replace static json imports with fetchData function
3. Fix any resulting errors from the content import

### CODE REVIEW

**Comment by SamiraMSadat** [comment]
@rayestrada Can you double check Delivery.jsx? I think the merge might have impacted the work you did for the A/B message, since the random function is no longer being used.

File: package.json
Comment (mlloydbixal) [inline package.json:1]: I don't think we ended up using this so it can be removed

File: src/useScript.js
Comment (SamiraMSadat) [inline src/useScript.js:1]: I forgot to remove this as well, it is not being used.

**Review by rayestrada — APPROVED** [review]

**Comment by rayestrada** [comment]
Superseded by #81

**Pull request closed without merging**